### ISSUE TITLE
error updating to TUF remote mirror: tuf: invalid key

### ISSUE THREAD

**Issue by ianlewis** [slsa-framework/slsa-verifier#325]
It seems that there is an error reading the TUF key when verifying signatures. This is occurring in v1.3.1 at least and is breaking the slsa-github-generator workflows at their latest version of v1.2.1 as well.

```
FAILED: SLSA verification failed: could not find a matching valid signature entry: got unexpected errors updating local metadata and targets: error updating to TUF remote mirror: tuf: invalid key
```

Not sure if this is a backwards incompatibility issue after a Rekor server upgrade or whether the TUF keys are just broken, but I assume it's the former?

/cc @asraa 

Related https://github.com/slsa-framework/slsa-github-generator/issues/1163

**Comment by ianlewis** [comment]
I wonder if it wasn't related to https://github.com/theupdateframework/go-tuf/pull/357

It seems to touch the affected areas

**Comment by asraa** [comment]
Thank you! Repeating some wording from the generator issue but just for clarity here:

This is because the Sigstore TUF root (that distributes all the key material for sigstore) had an update to meet TUF compliance which meant a change in key formats. Old versions of the TUF client library (go-tuf) do not understand the new compliant key format. 

We would be able to catch it if we can configure the generator and verifier against other TUF roots, and test other roots as well as catch the failure in pre-production at least (and hopefully staging): https://github.com/slsa-framework/slsa-github-generator/issues/387

**Comment by ianlewis** [comment]
Can we close this since we've released new versions of the verifier? Should we mark the old releases as unusable? We've gotten a lot of [push-back](https://github.com/slsa-framework/slsa-github-generator/issues/1253) about older releases of slsa-github-generator not being usable anymore so we've needed to mark them as clearly not usable (e.g. [v1.2.1](https://github.com/slsa-framework/slsa-github-generator/releases/tag/v1.2.1)).
